### DISCUSSION TITLE
[Projects Beta] Issue preview like old projects

### DISCUSSION THREAD

**Discussion by aBitEmberous** [community/community#8967] [Projects and Issues]
With the current Projects beta I feel like I am missing a key feature from the original Projects. In the old version you can click on an issue and see additional details, close it etc.

This is (in the old way) no longer possible with the Beta version. A new implementation of something like that would improve it.

**Comment by lars-reimann** [comment]
This is already on [their roadmap](https://github.com/github/roadmap/issues/287).
